stbt/keyboard.py

```diff
@@ -38,6 +38,12 @@ class Keyboard(object):
             W Q KEY_LEFT
             <etc>
 
+        For nodes that enter a character, use that character as the node name.
+        For the space-bar use SPACE. For other nodes that don't enter a
+        character when pressed use a descriptive name such as CLEAR or ENTER
+        (these nodes won't be used by ``enter_text`` but you can use them as a
+        target of ``navigate_to``).
```

Comment:
I'd rather have a concrete use case, otherwise how do we know what the structure should be? We can add a structured input in future in a backward-compatible way by checking if ``graph`` is a string or a list. For now I have made it accept a `networkx.DiGraph` so that you can construct it externally if you want to (generating a `DiGraph` instance might be easier than generating a string) but of course if you add any different attributes it won't make a difference because the implementation of `stbt.keyboard` doesn't know about them yet.

I do like this string format because it's simple, and the way you have to call `networkx.parse_edgelist` is non-trivial so I don't want to force user code to do that; so I've kept the string format too.